CMakeLists.txt

```diff
@@ -23,39 +23,47 @@ set(METAL_FILES csrc/mps_kernels.metal)
 # C++ sources are always included
 list(APPEND SRC_FILES ${CPP_FILES})
 
-set(COMPUTE_BACKEND "cpu" CACHE STRING "The compute backend to use (cpu, cuda, mps)")
+set(COMPUTE_BACKEND "" CACHE STRING "The compute backend to use (cpu, cuda, mps)")
 set_property(CACHE COMPUTE_BACKEND PROPERTY STRINGS cpu cuda mps)
 option(PTXAS_VERBOSE "Pass through -v flag to PTX Assembler" OFF)
 
 if(APPLE)
   set(CMAKE_OSX_DEPLOYMENT_TARGET 13.1)
 endif()
 
+option(BUILD_CUDA "Build bitsandbytes with CUDA support" OFF)
```

Comment:
This allows specifying both MPS and CUDA at the same time, which will fail as it is now. What is the expected outcome?

Reply:
previously implemented `CMakeLists.txt` does not do much and, `BUILD_MPS` even not work currently.